File: ESTUDO DE LINGUAGENS/PYTHON/Questões Uri/1116.py
qtd=int(input())
for x in range(0,qtd,1):
  valorX,valorY=map(int, input().split(" "))
  if valorY==0:
    print("divisao impossivel")
  elif (valorY!=0):
    soma=valorX/valorY
    print(soma)

File: ESTUDO DE LINGUAGENS/JAVASCRIPT/Questões uri/1116.js
let qtd=+lines.shift()
const pegarValoress = (line) => line.split(" ").map(a => +a)
for (let cont =0; cont<qtd; cont++){
        const [x,y] = pegarValoress(lines.shift())
        if (y==0){
                console.log("divisao impossivel")
        }
        else if (y!=0){
        let soma=x/y
        console.log(soma.toFixed(1))
        }
}
